config.cfg

```diff
@@ -0,0 +1,19 @@
+
+# Configuation file
+[app]
+DEBUG = True
+# This key will be regenerated and hidden at the end of this project
+SECRET_KEY = A0Zr98j/3yX R~XHH!jmN]LWX/,?RT
+SQLALCHEMY_DATABASE_URI = postgres://localhost/lemur
+
+[login_manager]
+SESSION_PROTECTION = strong
+LOGIN_VIEW = login
+
+[logger]
+FILEPATH = error_log/errors.log
+LEVEL = WARNING
+
+[class_time_info]
+CLASS_TIME = 16fall
+CURRENT_TIME = 16fall
```

Comment:
my argument is that they can simply set this value when the go to make a new class, and we can have a reasonable default computed from the current date. You don't need a separate page to fill this in. I guarantee professors will forget to update this value if it is buried in a config file on a server.